NEWS.md

```diff
@@ -59,7 +137,459 @@ Notable changes in Lepton EDA 1.9.5
+
+- The `makedepend` backend has been fixed and can now generate Make
+  rules for hierarchical schematics.  It can also work with
+  non-numbered schematic names.
+
+- The `spice-sdb` backend now emits verbose info to standard error, so
+  that verbose messages don't end up in the netlist.
+
+- The `vams` (Verilog AMS) backend now generates entity files
+  correctly.
+
+- The `redac` backend no longer duplicates connection info.
+
+- Several backends no longer emit trailing whitespace.
+
+- Changes for backend authors:
+
+  - A new `object-id` function has been added to allow backends to
+    access the object identifiers used during netlist generation.
+
+  - The `gnetlist-version` function has been removed.
+
+- Internal changes:
+
+  - A new Scheme test suite has been added, using the SRFI-64
+    framework.
+
+  - "Graphical" netlists are no longer used during netlist generation.
+
+  - Schematic page tree creation and transformation procedures are now
+    accessible from Scheme.
+
+  - A new `(gnetlist schematic)` module provides procedures for
+    working with schematics as first-class objects during netlisting.
+
+  - The new `(gnetlist option)` and `(gnetlist config)` modules
+    provide procedures for working with `gnetlist`'s command-line
+    arguments and configuration data.
+
+### Scheme API changes
+
+- A new procedure, `log!`, has been added to the `(geda log)` module.
+  It allows Scheme code to emit log messages in the same way that the
+  tools' C code does.
+
+- Two new Scheme procedures for working with source library have been
+  added to the `(geda library)` module: `source-library-contents`
+  and `set-source-library-contents!`.
+
+- Accessing Lepton values (objects, pages, etc.) via the Scheme API is
+  now much faster.
+
+Please check `info geda-scheme` for full details.
+
+### Changes when building from source:
+
+- Lepton requires Guile 2.0 or later for build.  However, running the
+  full test suite requires Guile 2.0.13.
+
+- The Lepton source tarball no longer includes `libintl`.  When
+  building Lepton with internationalisation support, GNU `gettext`
+  0.18 or later is required.
+
+- The configure script now supports an `--enable-contrib` switch.
+  Unless this is specified, the tools in the `contrib` directory will
+  not be compiled or installed.
+
+- The configure script now supports an `--enable-relocatable` switch.
+  When specified, Lepton tools do not have the configured prefix
+  compiled into them. This option is not supported on BSD systems.
+
+- The configure script now supports a `--disable-deprecated` switch.
+  When specified, various deprecated features and behaviour are
+  disabled, which may break backward-compatibility with some
+  configurations.
+
 ----------------------------------------------------------------
 
-Please see [docs/NEWS-1.9.4.md](docs/NEWS-1.9.4.md) for info on
-pre-1.9.5 changes.
+Please see [NEWS-1.9.2.txt](NEWS-1.9.2.txt) for info on
```

Comment:
Do you plan to move `NEWS-1.9.2.txt`? Otherwise, the hyperlink is broken.

Reply:
This seems to be fixed now, thanks!